taskcluster/ci/sign/kind.yml

```diff
@@ -0,0 +1,35 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I'd call the kind `signing` since that's the name we use in all the other taskgraph repos that have it. Will make it easier for folks to grep for and the like.